src/ublox/ubxGPS.cpp

```diff
@@ -990,7 +990,7 @@ bool ubloxGPS::parseHnrPvt( uint8_t chr )
         case 17:
         {
             ublox::hnr_pvt_t::flags_t flags = *((ublox::hnr_pvt_t::flags_t *) &chr);
-            m_fix.status = ublox::hnr_pvt_t::to_status( (ublox::hnr_pvt_t::status_t) m_fix.status, flags );
+            m_fix.status = ublox::hnr_pvt_t::to_status( (ublox::nav_status_t::status_t) m_fix.status, flags );
```

Comment:
Sorry, I just noticed this:

Based on the usage here, it appears that the first argument should be `gps_fix::status_t`, because case 16 above uses `parseFix` to set the `m_fix.status` member.  So `to_status` will be using the enum values from `gps_fix`, not from `nav_status`... (see next comment RE `to_status`)